editor/src/views/Crashes/CrashChangeLog.js

```diff
@@ -63,57 +71,79 @@ const isNewRecordEvent = change => change.operation_type === "create";
 
 /**
  * The primary UI component which renders the change log with clickable rows
+ * The Card defaults to being "collapsed", the open/closed state is stored in localStorage
  */
 export default function CrashChangeLog({ data }) {
   const [selectedChange, setSelectedChange] = useState(null);
+  const [isOpen, setIsOpen] = useState(
+    localStorage.getItem("showHistory") === "true"
+  );
 
   const changes = useChangeLogData(data);
   if (changes.length === 0) {
     return <p>No change history found</p>;
   }
 
+  const toggleCollapseHistory = () => {
+    const nextIsOpen = !isOpen;
+    setIsOpen(nextIsOpen);
+    localStorage.setItem("showHistory", nextIsOpen);
+  };
+
   return (
     <Card>
-      <CardHeader>Record history</CardHeader>
-      <CardBody>
-        <Table responsive striped hover>
-          <thead>
-            <tr>
-              <th>Record type</th>
-              <th>Event</th>
-              <th>Affected fields</th>
-              <th>Edited by</th>
-              <th>Date</th>
-            </tr>
-          </thead>
-          <tbody className="text-monospace">
-            {changes.map(change => (
-              <tr
-                key={change.id}
-                onClick={() => setSelectedChange(change)}
-                style={{ cursor: "pointer" }}
-              >
-                <td>{change.record_type}</td>
-                <td>{change.operation_type}</td>
-                <td>
-                  {isNewRecordEvent(change)
-                    ? ""
-                    : change.affected_fields.join(", ")}
-                </td>
-                <td>{change.created_by}</td>
-                <td>{formatDateTimeString(change.created_at)}</td>
+      <CardHeader>
+        <Button
+          block
+          color="link"
+          className="text-left m-0 p-0"
+          onClick={toggleCollapseHistory}
+        >
+          Record history
+          <Badge color="secondary float-right">{changes.length}</Badge>
```

Comment:
very nice touch 🙏 

how about we go a step further and add an icon and the same font size as the other card headers? i think a clock icon (`fa-clock-o`) works unless you have have a different FA v4 icon you prefer? https://fontawesome.com/v4/icons/

Reply:
I decided instead of clock to use `fa-history`

Reply:
🎯